tools/README.md

```diff
@@ -1,22 +1,18 @@
 # Snow Crash Development Tools
 
 ## Gyp
-Snow Crash uses (bundled) [Gyp](http://code.google.com/p/gyp/) build automation tool to generates its Makefiles. 
+Snow Crash uses (bundled) [Gyp](https://gyp.gsrc.io/) build automation tool to
+generate its Makefiles.
 
 ### Actual Bundled Version
-**r1846**
+[**SHA d84fed246cd42612c31a56a897e2d032e254a58a**](https://chromium.googlesource.com/external/gyp/+/d84fed246cd42612c31a56a897e2d032e254a58a)
```

Comment:
I don't like submodules with no control over the submodule repo. I think this is good enough.